src/test/java/Homework19_20.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19_20 extends BaseTest{
+    @Test
+
+    public void deletePlaylist () throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/650565cc-4c9b-4f86-bf9c-1615154a5aa2)